### ISSUE TITLE
the adversarial loss is different from the paper

### ISSUE THREAD

**Issue by fengxin619** [leftthomas/ImageDeblurring#20]
the keras edition use the loss from pix2pix , not the loss from the paper..

**Comment by nextfeixiag** [comment]
and i found some bug in the code.....
`# train adversarial net
            d_on_g_loss = d_on_g.train_on_batch(image_blur_batch, [1] * batch_size)
            print('batch %d d_on_g_loss : %f' % (index + 1, d_on_g_loss))

            # train generator
            g_loss = g.train_on_batch(image_blur_batch, image_full_batch)
            print('batch %d g_loss : %f' % (index + 1, g_loss))`

**Comment by leftthomas** [comment]
@nextfeixiag you could change that as the paper said, I used this just to experiment
